Formula/migrate.rb

```diff
@@ -0,0 +1,16 @@
+class Migrate < Formula
+  desc "Database migrations. CLI and Golang library."
+  homepage "https://github.com/mattes/migrate"
+  url "https://github.com/mattes/migrate/releases/download/v3.0.1/migrate.darwin-amd64.tar.gz"
+  version "3.0.1"
+  sha256 "17e19678e930d4a48114937fb07300c107088d86a80c798630fabc08d32ee4ee"
+
+  def install
+    mv "migrate.darwin-amd64", "migrate"
```

Comment:
[Definitely not](https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md#we-dont-like-binary-formulae) :)

> Our policy is that formulae in the core tap (homebrew/core) must be open-source and either built from source or produce cross-platform binaries (e.g. Java). Binary-only formulae should go to Homebrew Cask.